chapters/appendix.adoc

```diff
@@ -60,7 +60,7 @@ Chapter <<Offenses>>, section <<Minor Offenses>>:
 | Event | Applicability | Command | AutoRef
 
 | <<Aimless Kick [small]#(_division B only_)#, Aimless Kick>> | <<Ball In And Out Of Play, ball in play>> | <<Stop>>, then <<Indirect Free Kick>> | icon:check[role="green"]
-| <<Lack Of Progress>> | <<Ball In And Out Of Play, ball in play>> | <<Stop>>, then <<Indirect Free Kick>> | icon:check[role="green"]
+| <<Lack Of Progress>> | <<Ball In And Out Of Play, ball in play>> | <<Stop>>, then <<Force Start>> | icon:check[role="green"]
```

Comment:
Stop, then Force Start implies that both teams have to keep distance before force-start can be given. If this rule is not applied here, that should be worded differently. @g3force did I understand you correctly that the improved speed should be achieved by not stopping the game? If so, it might be an option to remove "lack of progress" in the case of freekicks and instead state in "ball goes into play" that the ball goes into play at latest 5 seconds after the Indirect / Direct command was issued. This results in slightly different rules (Force-Start is given vs. teams have to count and wait on their own), but might be the more straight-forward way of implementing this change.

Reply:
I really like this idea. The teams do not even need to count themselves, because the referee messages already contain the counter since 2019. The only disadvantage is, that teams actually have to modify their system slightly, but that should be fine, as the change is small and not mandatory to confirm to the rules.